### PR TITLE
noobaa/core: Modify debug module to chnge log color

### PR DESCRIPTION
**Pull request by aspandey**
We need to provide an options so that a user can disable/enable color of debug logs generated by core, endpoint and backing store.

### Fixes:
- Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2297435

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -480,6 +480,8 @@ config.EVENT_LEVEL = 5;
 config.LOG_TO_STDERR_ENABLED = true;
 config.LOG_TO_SYSLOG_ENABLED = false;
 
+config.LOG_COLOR_ENABLED = true;
```
Comment (liranmauda) [inline config.js:483]: I did not see any use of this config, Do we need it?

File: src/util/debug_module.js [hunk 2]
```diff
@@ -289,7 +290,45 @@ class InternalDebugLogger {
         return tmp_mod.__level;
     }
 
+    message_format_without_color(level, args) {
```
Comment (liranmauda) [inline src/util/debug_module.js:293]: I do not think we need 2 functions, we can set the `level_color` in the orig function to be an empty string, and check if we want colors or not.

**Review by liranmauda — CHANGES_REQUESTED** [review]

File: config.js (hunk 1 above)
Comment (aspandey) [reply config.js]: done

File: src/util/debug_module.js (hunk 2 above)
Comment (aspandey) [reply src/util/debug_module.js:293]: done

**Review by liranmauda — APPROVED** [review]

**Pull request merged**